crankshaft-docker/src/lib.rs

```diff
@@ -28,6 +28,9 @@ pub enum Error {
     /// A required value was missing for a builder field.
     #[error("missing required builder field `{0}`")]
     MissingBuilderField(&'static str),
+    /// An operation timed out.
+    #[error("an operation timed out: {0}")]
+    Timeout(String),
```

Comment:
Just a note to remove this variant if we end up removing the `default_timeout` function.